src/Symfony/Bundle/TwigBundle/Loader/FilesystemLoader.php

```diff
@@ -89,7 +89,14 @@ protected function findTemplate($template, $throw = true)
         }
 
         if (false === $file || null === $file) {
-            throw new \Twig_Error_Loader(sprintf('Unable to find template "%s".', $logicalName), -1, null, $previous);
+            try {
+                list($namespace, $name) = $this->parseName($logicalName);
```

Comment:
```parseName()``` became private in [twig master](https://github.com/twigphp/Twig/blob/master/lib/Twig/Loader/Filesystem.php#L230) and as I see, this is the problem in travis build too:

```
Fatal error: Call to private method Twig_Loader_Filesystem::parseName() from context
```